### PR TITLE
[HOPSWORKS-2794]Use templates for code indices and enable expat reindexing

### PR DESCRIPTION
**Pull request by o-alex**

### CODE REVIEW

File: recipes/default.rb [hunk 1]
```diff
@@ -84,13 +90,32 @@
...
+  user 'root'
+  code <<-EOH
+    set -e
+    mv -f #{node['elastic']['backup_dir']}/* #{node['elastic']['data_volume']['backup_dir']}
+    mv -f #{node['elastic']['backup_dir']} #{node['elastic']['backup_dir']}_deprecated
+  EOH
+  only_if { conda_helpers.is_upgrade }
+  only_if { File.directory?(node['elastic']['backup_dir'])}
+  not_if { File.symlink?(node['elastic']['backup_dir'])}
+end
```
Comment (SirOibaf) [inline recipes/default.rb:103]: You don't need to move this - as the backup directory is something that we just created, it doesn't exists on existing clusters.

**Review by SirOibaf — CHANGES_REQUESTED** [review]

File: recipes/default.rb (hunk 1 above)
Comment (o-alex) [reply recipes/default.rb]: ok, removed this part.

**Review by SirOibaf — APPROVED** [review]

**Review by maismail — CHANGES_REQUESTED** [review]
@o-alex can you also remove the (reindex-base-indices.sh.erb) file and its references as part of your PR, basically revert the following commit 

https://github.com/logicalclocks/elasticsearch-chef/commit/289b7d1a1ebe294e6f23b1a982c4181bcd4b360f

**Comment by o-alex** [comment]
@maismail reverted that PR and removed some variables as well that were there just for the script itself

**Review by maismail — APPROVED** [review]

**Pull request merged**